.github/workflows/codeception.yml

```
@@ -0,0 +1,78 @@
+name: "Codeception Tests"
+
+on:
+    schedule:
+        -   cron: '0 3 * * 1,3,5'
+    pull_request:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+    push:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+            - "*_actions"
+
+env:
+    PIMCORE_PROJECT_ROOT: ${{ github.workspace }}
+    APP_ENV: test
+    PIMCORE_TEST: 1
+    PIMCORE_TEST_DB_DSN: "mysql://root@127.0.0.1:33006/pimcore_test"
+
+jobs:
+    codeception-tests:
+        name: "Codeception tests"
+        runs-on: "ubuntu-20.04"
+        continue-on-error: ${{ matrix.experimental }}
+        env:
+            PIMCORE_TEST_DB_DSN: "mysql://root@127.0.0.1:33006/pimcore_test?serverVersion=${{ matrix.server_version }}"
+        strategy:
+            matrix:
+                include:
+                    - { php-version: 8.1, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: highest, experimental: false }
```

```suggestion
                    - { php-version: 8.1, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: lowest, experimental: false }
```
Run at-least one against lowest